apps/mobile/src/components/ToastNotification/index.tsx

```diff
@@ -31,8 +33,8 @@ function ToastNotification(): JSX.Element {
 
   return (
     // this view has to be present in tree
-    // without it existing animation on Toast will not work
-    <Stack pos="absolute" alignSelf="center" top={-10}>
```

Comment:
Let's not change this. But I would prefer to not use the shortcuts. The code is more readable that way. But it's up for the discusion